hir/src/asm/isa.rs

```diff
@@ -1781,17 +1771,25 @@ impl MasmOp {
             Self::U32Clo => Instruction::U32Clo,
             Self::U32Cto => Instruction::U32Cto,
             Self::U32Lt => Instruction::U32Lt,
-            Self::U32LtImm(imm) => Instruction::U32LtImm(imm.into()),
+            Self::U32LtImm(imm) => return smallvec![Instruction::PushU32(imm), Instruction::U32Lt],
```

Comment:
Just a heads up. I removed immediates variant of these ops in my VM update PR https://github.com/0xPolygonMiden/compiler/pull/261

Reply:
I'd prefer to keep them in our IR, even though they are gone from the underlying MASM AST, because it is easier to canonicalize to that form when lowering from HIR, and then do peephole optimization based on that canonical form. We don't do that yet, but will start doing so pretty soon.

Reply:
Oh, I meant I removed the VM's counterparts, not ours. The changes in my PR are exactly as you made here. Sorry for the confusion.

Reply:
Oh I see, my bad!